qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -684,3 +740,41 @@ QImage HOCRPdfExporter::getSelection(const QRect& bbox)
 {
 	return m_displayerTool->getSelection(bbox);
 }
+
+void HOCRPdfExporter::paperSizeChanged() {
+	PaperSizeMetrics currentMetric = static_cast<PaperSizeMetrics>(ui.comboBoxPaperSizeMetric->itemData
+			(ui.comboBoxPaperSizeMetric->currentIndex()).toInt());
+
+	auto textPaperSize = ui.comboBoxPaperSize->itemData(ui.comboBoxPaperSize->currentIndex()).toString();
+	if(textPaperSize == "custom") {
+		ui.lineEditPaperSizeWidth->setEnabled(true);
+		ui.lineEditPaperSizeHeight->setEnabled(true);
+	} else if(textPaperSize == "source") {
+		ui.lineEditPaperSizeWidth->setText("");
+		ui.lineEditPaperSizeHeight->setText("");
```

Comment:
Doesn't this make the user potentially loose his previously entered custom width/height? I'd just hide the line edits when mode is "source".

Reply:
How can i hide line edits? Which method?

Reply:
setVisible(false)?

Reply:
Fixed.